src/services/authServices.tsx

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import {
+  AUTH_TOKEN_KEY,
+  REFRESH_TOKEN_KEY,
+  SERVER_BASE_URL,
+} from "../constants/constants";
+
+export const handleLogin = async (username: string, password: string) => {
+  try {
+    const response = await axios.post(`${SERVER_BASE_URL}/login`, {
+      username,
```

Comment:
Keep API calls in a service file and handle token management in a separate file to keep responsibilities clear and organized.

Follow the principle of separation of concern